.github/workflows/diff.yml

```diff
@@ -0,0 +1,84 @@
+name: 'Create ROBOT diffs on Pull requests'
+
+on:
+  # Triggers the workflow on pull request events for the master branch
+  pull_request:
+    branches: [ master ]
+    paths:
+      - 'src/ontology/uberon-edit.obo'
```

Comment:
I assume this means it only runs when that file is edited? Wonder if we should keep this or get it to run every time (given it runs on reasoned file, albeit -simple, we might want to run this every time so that when other files that change that affects it, this will show). Thoughts @matentzn?

Reply:
We could add all the files that can potentially affect the diff in here.. Remeber this runs for quite long - so would be better if we could somehow rain in the runtime when people just change the docs..

Reply:
ill run a test to make sure everything works well :)

Reply:
sweet! works nicely! thanks @ubyndr! This will be SO helpful! 
btws took `1h 13m 47s` to run :D